cleanlab_studio/studio/trustworthy_language_model.py

```diff
@@ -83,26 +84,29 @@ def __init__(
 
     def batch_prompt(
```

Comment:
I was planning on leaving in for reverse compatibility. I can add a deprecation warning to the docstring if that's helpful, and we'll update our tutorials to point to the `prompt` method instead of `batch_prompt`, but I'm not sure that removing these is the right move.

Reply:
dont need reverse compatibility, nobody has evidently been able to use batch_prompt and batch_confidence_score yet

We never want to offer multiple ways of doing the same thing that are user facing, it detracts from user experience.

Reply:
We've had the researched at GA tech use the batch APIs, but we can drop them to private if you want